packages/executor/src/execution/execute.ts

```diff
@@ -1550,7 +1550,8 @@ export function flattenIncrementalResults<TData>(
   let done = false;
   signal?.addEventListener('abort', () => {
     done = true;
-    subsequentIterator.throw?.(signal?.reason);
+    // we catch here to avoid unhandled promise rejection
+    subsequentIterator.throw?.(signal?.reason).catch(() => undefined);
```

Comment:
Since the same error is already raised from the AsyncIterable returned by `normalizedExecutor` it is safe to ignore it here.